src/map/entities/charentity.h

```diff
@@ -511,6 +511,9 @@ class CCharEntity : public CBattleEntity
     bool getBlockingAid() const;
     void setBlockingAid(bool isBlockingAid);
 
+    // Send updates about dirty containers in post tick
+    std::map<uint8, bool> dirtyInventoryContainers;
```

Comment:
You could replace the uint8 with CONTAINER_ID

Reply:
done